cpp/modmesh/view/RMainWindow.cpp

```diff
@@ -40,6 +40,10 @@ void RMainWindow::setUp()
     m_pytext->setAllowedAreas(Qt::LeftDockWidgetArea | Qt::RightDockWidgetArea);
     addDockWidget(Qt::RightDockWidgetArea, m_pytext);
 
+    m_pycon = new RPythonConsoleDockWidget(QString("Console"), this);
+    m_pycon->setAllowedAreas(Qt::BottomDockWidgetArea | Qt::TopDockWidgetArea);
```

Comment:
Allow the console to be at the bottom or the top.  Having it in the left or right could make it too narrow.